benchmarks-api/src/main/java/io/scalecube/benchmarks/BenchmarksSettings.java

```diff
@@ -175,9 +207,14 @@ private String minifyClassName(String className) {
     private TimeUnit rateUnit = RATE_UNIT;
     private long numOfIterations = NUM_OF_ITERATIONS;
     private Duration rampUpDuration = RAMP_UP_DURATION;
-    private Duration rampUpInterval = RAMP_UP_INTERVAL;
     private boolean consoleReporterEnabled = CONSOLE_REPORTER_ENABLED;
+    private int users;
+    private int messageRate;
     private String[] args = new String[] {};
+    // dynamic params
```

Comment:
Could you plz move this ``private String[] args = new String[] {};``   above [line with users/injectors](https://github.com/scalecube/scalecube-benchmarks/pull/66/files#diff-4500e663c0f78a27c0d3c08ba9733d67R211)